src/pages/workspace/reimburse/WorkspaceReimburseNoVBAView.js

```diff
@@ -116,6 +102,32 @@ class WorkspaceReimburseNoVBAView extends React.Component {
         }, null);
     }
 
+    handleKeyPress = (event) => {
```

Comment:
No inline functions.  Function names should reflect what they do instead of where they are used. 
```suggestion
    debounceUpdateOnCursorMove(event) {
```

Reply:
Done

Reply:
Sorry, I don't this change. Did you push it?